### DISCUSSION TITLE
Map component in VF

### DISCUSSION THREAD

**Discussion by sandykadam** [visual-framework/vf-core#1410] [Proposals]
Hi Guys,

It will be great if we can have map component in VF which aligns to EBI/EMBL theme, right now we using map only on live map i.e https://www.ebi.ac.uk/web/livemap/ but recently there is been requests on different project to show map like Covidportal, Training website. So its a good opportunity to have component and user can customised it based on their needs.

**Comment by khawkins98** [comment]
This is a good idea.

The current map uses MapBox (and our keys), so we'd want to be conscious to limit it to EBI and EMBL use only.

I think it might be bigger than "just a component" and some kind of mini project/boilerplate.

We'd need to prioritise the work, and at the moment it's a bit down the list.

**Reply by sturobson** [reply]
> I think it might be bigger than "just a component"

Looking at the existing usage - it aligns with the larger 'container' ideal we're going for.

**Comment by khawkins98** [comment]
A note: Design and International Relationship are also keenly interested in some reusable tooling here.
